lib/api_hammer/active_record_cache_find_by.rb

```diff
@@ -124,8 +125,8 @@ def cache_key_for(find_attributes)
     def flush_find_cache
       self.class.send(:cache_find_bys).each do |attribute_names|
         find_attributes = attribute_names.map { |attr_name| [attr_name, attribute_was(attr_name)] }
-        self.class.instance_exec(find_attributes) do |find_attributes|
-          finder_cache.delete(cache_key_for(find_attributes))
+        self.class.instance_exec(find_attributes) do |find_attributes_|
+          finder_cache.delete(cache_key_for(find_attributes_))
```

Comment:
warning about find_attributes block variable shadowing local. which it doesn't actually because that's not how instance_exec works, but this change is just to silence the warning.